lib/index.js

```diff
@@ -180,6 +180,28 @@ export const Dataviz = function (options = {}) {
     this.config.stacked = true;
   }
 
+  // overwriting options for sparkline chart
+  if (this.config.type === 'sparkline') {
```

Comment:
let's make it more universal - like you did for range charts line :166 
`.includes('-sparkline')`